src/server/static/js/lines.js

```diff
@@ -41,69 +41,72 @@ function getCountries(data) {
   return [...new Set(data.map(d => d.Country))];
 }
 
-function getCountryBetaData(data) {
-  result = [];
-  let combinations = _.uniqBy(data, r =>
-    [r.Country, r.Mitigation].join()
-  ).map(r => ({ country: r.Country, beta: r.Mitigation, items: [] }));
-  data.forEach(row => {
-    _.find(combinations, {
-      country: row.Country,
-      beta: row.Mitigation
-    }).items.push([
-      today.addDays(row.Timestep),
-      row["Cumulative Median_s=0.85_at=0.2"],
-      row["Cumulative Median_s=0.7_at=0.7"],
-      row["Cumulative Median_s=0.1_at=0.2"],
-      row["Cumulative Median_s=0.85_at=0.7"],
-      row["Cumulative Median_s=0.1_at=0.7"],
-      row["Cumulative Median_s=0.7_at=0.2"]
-    ]);
-  });
-  return combinations;
-}
+// function getCountryBetaData(data) {
```

Comment:
Please delete old code, don't just comment it out. This is what version control is for.